tests_tf/test_utils.py

```diff
@@ -79,6 +81,23 @@ def test_other_classes_return_val(self):
         res_expected = [0, 1, 3, 4]
         self.assertTrue(res == res_expected)
 
+    def test_get_logits_over_interval(self):
+        import tensorflow as tf
+        model = cnn_model()
+        wrap = KerasModelWrapper(model)
+        x = tf.placeholder(dtype=tf.float32, shape=(1, 28, 28, 1))
+        adv_x = x
+        img = np.ones(shape=(1, 28, 28, 1))
+        with tf.Session() as sess:
+            tf.global_variables_initializer().run()
+            log_prob = utils.get_logits_over_interval(sess, wrap,
+                                                      x, adv_x, img,
+                                                      min_epsilon=-10,
+                                                      max_epsilon=10,
+                                                      num_points=21)
+            log_prob -= log_prob[0]
+            self.assertTrue(np.count_nonzero(log_prob) == 0)
```

Comment:
better to use `assertEqual` so that if  this fails, the travis log will show how many were nonzero